agent.go

```diff
@@ -63,9 +64,13 @@ type Payload struct {
 
 // ProbeResult structure for network probing results
 type ProbeResult struct {
```

Comment:
I suppose it also should be stored via Kubernetes API. In such case u need add additionl endpoint for it